README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Build Status](https://travis-ci.org/marius92mc/github-stargazers.svg?branch=master)](https://travis-ci.org/marius92mc/github-stargazers)
 [![PyPI version](https://badge.fury.io/py/github-stargazers.svg)](https://badge.fury.io/py/github-stargazers)
+[![tweet](http://jpillora.com/github-twitter-button/img/tweet.png)](https://twitter.com/intent/tweet?text=github-stargazers+on+PyPi!&url=https%3A%2F%2Fgithub.com%2Fmarius92mc%2Fgithub-stargazers&hashtags=github&original_referer=http%3A%2F%2Fgithub.com%2F&tw_p=tweetbuttonw_p=tweetbutton)
```

Comment:
I used [this](http://jpillora.com/github-badge-maker/) friendlier way to fill the _text_ that will appear as the tweet default message. What do you think it might be a better content for it instead of _"github-stargazers on PyPi!"_, @ignacio-chiazzo?